frontend/apps/marketing/src/components/iframe/iframeContentfulDefinition.ts

```diff
@@ -0,0 +1,48 @@
+// Creates a definition for the Iframe component to be used in Contentful Studio
+import {ComponentDefinition} from '@contentful/experiences-sdk-react';
+
+export const IframeContentfulComponentDefinition: ComponentDefinition = {
+  id: 'iframe',
+  name: 'iFrame',
+  category: 'Advanced',
+  thumbnailUrl:
+    'https://images.ctfassets.net/90t6bu6vlf76/1qy9FC9Bqb4ADrpyszIa5M/5e791afb110fc5bc8caa95710c1dea16/component_code_thumbnail.png',
```

Comment:
Super nit: rename this image file in Contentful (and here) to be consistent
```suggestion
    'https://images.ctfassets.net/90t6bu6vlf76/1qy9FC9Bqb4ADrpyszIa5M/5e791afb110fc5bc8caa95710c1dea16/component_iframe_thumbnail.png',
```